gunfolds/utils/clingo.py

```diff
@@ -105,26 +125,35 @@ def clingo(command, exact=True,
         trendy: Use defaults geared towards industrial problems
     :type configuration: string
 
-    :param cpath: clingo path
-    :type cpath: string
+    :param optim: a list containing configuration for optimization algorithm and optionally a bound  [<arg>, <bound>]
+        <arg>: <mode {opt|enum|optN|ignore}>[,<bound>...]
+        opt   : Find optimal model
+        enum  : Find models with costs <= <bound>
+        optN  : Find optimum, then enumerate optimal models
+        ignore: Ignore optimize statements
+        <bound> : Set initial bound for objective function(s)
+    :type optim: list
 
     :param pnum: number of parallel threads to run clingo on
     :type pnum: integer
 
     :returns: results of parsed equivalent class
     :rtype: dictionary
     """
+    if optim is None:
+        optim = ['optN']
```

Comment:
why not make `['optN']` the default?

Reply:
see above comment

Reply:
I addressed the above with a much simpler solution:

1. a sensible default `"optN"`
2. let the user pass the parameters as a string with a comma separated parameter if needed
3. make an assert that checks if the user passed something longer than 2 parameters